### PR TITLE
Fixes #19 - added Dockerfile and updated README for building and running the container

### LINKED ISSUE #19: Create docker container for maestro service
Create a Dockerfile to dockerize the maestro service.

### PR DESCRIPTION
**Pull request by navarroc**

### CODE REVIEW

**Comment by navarroc** [comment]
This just adds the Dockerfile and commands to build/run the container. I believe @ywkim312 was going to handle the github action to build/push the image as a separate issue.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM python:3.9-slim
+
+ARG WORKERS
+ENV WORKERS=${WORKERS:-4}
+
+ENV POETRY_VIRTUALENVS_CREATE false
+
```
Comment (ywkim312) [inline Dockerfile:7]: This might not be needed now but adding some postgresql connection parameters needs to be imbedded as ENV in the future.

**Review by ywkim312 — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (navarroc) [reply Dockerfile:7]: I pass those in through the environment file, but we could add them as ENV explicitly in the Dockerfile too

File: README.md [hunk 2]
```diff
@@ -33,3 +33,16 @@ Run:
...
+
+### Build Docker container
+
+docker build --tag incore/maestro-service:latest .
+
+### Run Docker container
+See Setup Environment section for populating the example-env
+
+To run the container:
+docker run -p 8000:8000 --env-file env-example incore/maestro-service:latest
```
Comment (longshuicy) [inline README.md:47]: The env-example still has `direct4ag` could we change that database name to `maestro`?
also could we just drop the test database?

```
POSTGRES_SERVER=localhost
POSTGRES_PORT=5432
POSTGRES_USER=postgres
POSTGRES_PASSWORD=
POSTGRES_DB=direct4ag
POSTGRES_TEST_DB=tests_direct4ag
```

File: Dockerfile (hunk 1 above)
Comment (ywkim312) [reply Dockerfile:7]: Yes we need it for deploying it into k8s

File: README.md (hunk 2 above)
Comment (longshuicy) [reply README.md:47]: Also can we add the Router prefix to the env file?
```
export ROUTER_PREFIX=/maestro
```

Comment (navarroc) [reply README.md:47]: Yes, I can update the env-example.

**Review by longshuicy — APPROVED** [review]

**Comment by navarroc** [comment]
I will update to use ENV before merging this

**Comment by navarroc** [comment]
@ywkim312 Can you check the ENV variables I added and let me know if this is sufficient? If not, please suggest changes so I can update the PR.

**Pull request merged**